app/views/curation_concerns/base/_form_relationships.html.erb

```diff
@@ -1,4 +1,9 @@
-<h3>This Work in Collections</h3>
+<%
+  batch = current_page?(sufia.new_batch_upload_path)
+  in_collection_label = "This Work in Collections"
+  in_collection_label = "These New Works in Collections" if batch
+%>
+<h3><%= in_collection_label %></h3>
```

Comment:
Or can we handle this via i18n (vs. having logic in the view) like here? https://github.com/projecthydra/sufia/blob/master/app/views/curation_concerns/base/new.html.erb#L3

I'd also recommend re-wording a bit. Why would one label use the word "new" and the other not? Suggest striking it from both. Thoughts, @projecthydra/sufia-ui-ux-advisors?